app/models/knight.rb

```diff
@@ -2,4 +2,12 @@ class Knight < Piece
   def to_unicode
     black? ? "\u265E" : "\u2658"
   end
+
+  def valid_move?(destination_x, destination_y)
+    distance_x = (destination_x - x_position).abs
+    distance_y = (destination_y - y_position).abs
+
+    return true if ((distance_x - distance_y).abs == 1) && (distance_x > 0) && (distance_x < 3) && (distance_y > 0) && (distance_y < 3)
```

Comment:
I'm curious but what would happen if you simplify this line like this (assuming it is doing the same thing) `return true if distance_x == 1 && distance_y == 2 || distance_x == 2 && distance_y == 1`